project.yaml

```diff
@@ -5,6 +5,8 @@ runner:
   node:
     name: "@subql/node"
     version: "*"
+    options:
+      historical: false
```

Comment:
Out of curiosity what does this mean?

Reply:
This mean the subquery won't create historical tables based on the blocks. This is a indexing feature to filter information by block, creating additional tables and inner relations out of the created schemas, this helps to find info by block. e.x "Get transfers by block number N" We do not need that, plus, it was making the indexing process slower (one of the reasons could say)